health-claim/src/components/Login/RegisterForm.js

```diff
@@ -17,10 +21,23 @@ export default class RegisterForm extends Component {
 
   handleSubmit(event) {
     this.props.register(this.state.username, this.state.password);
+    console.log(this.state.username);
     event.stopPropagation();
     event.preventDefault();
   }
 
+  handleClick(compName, e) {
+    console.log(compName);
+    this.setState({ render: compName });
+  }
+
+  _renderSubComp() {
+    switch (this.state.render) {
+      case "QRReader":
```

Comment:
Why do we keep this as a string and not as a bool?

Reply:
Ok, good point. I simplified